Resources/Prototypes/Entities/Objects/Weapons/Melee/stunprod.yml

```diff
@@ -0,0 +1,49 @@
+- type: entity
+  name: stun prod
+  parent: BaseItem
+  id: Stunprod
+  description: A stun prod for illegal incapacitation.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/stunprod.rsi
+    netsync: false
+    layers:
+    - state: stunprod_off
+      map: [ "enum.ToggleVisuals.Layer" ]
+  - type: Stunbaton
+    energyPerUse: 100
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 9
+    bluntStaminaDamageFactor: 1.5
+    angle: 60
+    animation: WeaponArcThrust
+  - type: StaminaDamageOnHit
+    damage: 30
```

Comment:
This is still going to do as much stamina damage as a stun baton when it should be at least 5 or more hits.

30 + 13.5 = 43.5 so stun in 3 hits easily.